api/soap/mantisconnect.wsdl

```diff
@@ -747,6 +814,30 @@
   <part name="per_page" type="xsd:integer" /></message>
 <message name="mc_filter_get_issue_headersResponse">
   <part name="return" type="tns:IssueHeaderDataArray" /></message>
+<message name="mc_filter_search_issue_headersRequest">
+  <part name="username" type="xsd:string" />
+  <part name="password" type="xsd:string" />
+  <part name="filter" type="tns:FilterSearchData" />
+  <part name="page_number" type="xsd:integer" />
```

Comment:
Would it make sense to include page_number and per_page in the FilterSearchData? TBH I'm not sold on the idea but putting it up for discussion